_content/setup-wallets-commandline.md

```diff
@@ -0,0 +1,141 @@
+## Interledger Protocol (ILP) TestNet: Getting Started
+
+This tutorial describes how to use the command line and test XRP to:
+
+1. Create an account in the Xpring.io Interledger Testnet.
+2. Add test XRP to your account using the TestNet Rainmaker \(our version of a "faucet"\).
+3. Check your account balance.
+4. Send test XRP to another payment pointer.
+5. Receive test XRP.
+
+To do the same with a GUI-based interface, refer to [setup-wallets.md].
+
+### 1. Create an account
+
+Create a new account with this command:
```

Comment:
Would it be worth making articulating this in a little more generic way? e.g. "Send an HTTP request to create a new account. For this tutorial, we'll use `curl` on the command line to make requests to the API, but you can use an HTTP client library in any language. Here's how to create an account on the command line:"